### ISSUE TITLE
dúvida

### ISSUE THREAD

**Issue by rheydner** [jvdutra/open-source-rpg#7]
ó precisa dar “npm install && npm start” ?

**Comment by Luccaop** [comment]
Sim, antes você precisa ter o Node instalado e o npm (ele vem junto com o Node).
O npm install serve para baixar as dependências usadas no projeto, portanto você só ira usar da 1° vez que iniciar o projeto (ou no proximo update).
O npm start serve para iniciar o servidor.

**Comment by Nicodemos234** [comment]
Você também vai precisar de um servidor de banco de dados mysql (ou algum do tipo), rodando e configurar o .env com base no .env.template, e também rodar as migrations do prisma! Pelo menos eu precisei para funcionar direito aqui.
